openstf/model/objective.py

```diff
@@ -31,19 +33,83 @@ class RegressorObjective:
         study.optimize(objective)
     """
 
+    def process_tuple(
+        self,
+        trial: optuna.trial.FrozenTrial,
+        key: str,
+        value: Tuple[Union[int, float], Union[int, float]],
+    ) -> Union[int, float]:
+        """Pick hyperparameter for tuple"""
+        tup, log = value
+        start_value, end_value = tup
+
+        error = "input values for the tuple can be float/float, float/int, int/float, int/int"
+
+        if isinstance(start_value, float) or isinstance(end_value, float):
+            return trial.suggest_float(
+                key, float(start_value), float(end_value), log=log
+            )
+
+        elif isinstance(start_value, int) and isinstance(end_value, int):
+            return trial.suggest_int(key, start_value, end_value)
+
+        else:
+            raise TypeError(error)
+
+    def get_trial_parameters(
+        self, model_params: dict, trial: optuna.trial.FrozenTrial
+    ) -> dict:
+        """
```

Comment:
Nice to add a docstring! A couple minor remarks. The title should go on a single line and an additional description below. Furthermore you could also describe the arguments and return optionally.

In your description you refer to "the dictionairy" its more clear to name the argument: `model_params` can contain etc. 

Also a dictionairy had key values, you are probably talking about the values in the dictionairy?

Also some lines seem to be over 88 chars. Our formatter black can do a lot but it does not split docstring or comments.